docker/release/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG PYTORCH="1.8.1"
+ARG CUDA="10.2"
+ARG CUDNN="7"
+
+FROM pytorch/pytorch:${PYTORCH}-cuda${CUDA}-cudnn${CUDNN}-devel
+
+# To fix GPG key error when running apt-get update
+RUN rm /etc/apt/sources.list.d/cuda.list
```

Comment:
May consider running the commands in line 8-11 in a single RUN, since each RUN command essentially builds a new layer and these four commands should be combined together to work.

Reply:
Okay